demo/index.html

```diff
@@ -48,8 +48,8 @@
           var combobox = document.querySelector('#demo2');
           combobox.items = elements;
 
-          combobox.addEventListener('value-changed', function() {
-            document.querySelector('#selected-value').innerHTML = combobox.value;
+          combobox.addEventListener('value-changed', function(e) {
+            document.querySelector('#selected-value').innerHTML = e.target.value;
```

Comment:
> IMO, combobox.value is easier to get then e.target.value

Without doubts, it is easier, but I don't have a better solution here except adding `var combobox = e.target;` before that line which using `combobox.value`. Thoughts?

> One example: 264b4ac#diff-25dd8f443c3ef3c623850d3a204e916c

Well, adding numbers to variables isn't a good way of naming. Example: I'm adding the new example between 2dn and 3rd examples. If I follow the `comboboxN` naming convention, then I will need to rename all examples from 3rd to the last. Manually :)